### DISCUSSION TITLE
What's "Video Bus"?

### DISCUSSION THREAD

**Discussion by HubKing** [sezanzeb/input-remapper#811] [Q&A]
I am using a laptop, and I can understand other items in the device list but I don't know what "video bus" is. How is this an input device?

![image](https://github.com/sezanzeb/input-remapper/assets/10900489/9103a9c1-b5ec-4791-9d9d-143f67cc69f2)

**Comment by neyel8r** [comment]
same here... not sure why this is showing up as an input device. is there some way to remove it?

i have a "Sleep Button" device on my main screen as well

**Comment by yangu-hury** [comment]
The same. I'm really curious about it... I haven't the courage to try it, because I feel It will crash Xorg. When I got some time I will probably check the source for it.
